bin/add-pr-comment.sh

```diff
@@ -48,16 +69,29 @@ if [[ -z "${prNumber}" ]]; then
 	exit 1
 fi
 
+if [[ ${operation} == "delete" ]]; then
+	if [[ -z "$watermarkName" ]]; then
+		echo "A watermark is required for deleting comments."
+		exit 1
+	fi
+	watermark="<!--${watermarkName}-watermark:apr@v1-->"
+	commentId=$(get_existing_comment $prNumber $watermark)
```

Comment:
`$prNumber` and `$watermark` should be quoted to avoid globbing and word splitting (https://github.com/koalaman/shellcheck/wiki/SC2086). Found via https://www.shellcheck.net/